src/Tizen.NUI/src/internal/RenderTask.cs

```diff
@@ -48,96 +48,96 @@ public static RenderTask GetRenderTaskFromPtr(global::System.IntPtr cPtr)
 
         internal class Property
         {
-            internal static readonly int VIEWPORT_POSITION = Interop.RenderTask.RenderTask_Property_VIEWPORT_POSITION_get();
-            internal static readonly int VIEWPORT_SIZE = Interop.RenderTask.RenderTask_Property_VIEWPORT_SIZE_get();
-            internal static readonly int CLEAR_COLOR = Interop.RenderTask.RenderTask_Property_CLEAR_COLOR_get();
-            internal static readonly int REQUIRES_SYNC = Interop.RenderTask.RenderTask_Property_REQUIRES_SYNC_get();
+            internal static readonly int ViewportPosition = Interop.RenderTask.RenderTaskPropertyViewportPositionGet();
+            internal static readonly int ViewportSize = Interop.RenderTask.RenderTaskPropertyViewportSizeGet();
+            internal static readonly int ClearColor = Interop.RenderTask.RenderTaskPropertyClearColorGet();
+            internal static readonly int RequiresSync = Interop.RenderTask.RenderTaskPropertyRequiresSyncGet();
         }
 
-        internal static SWIGTYPE_p_f_r_Dali__Vector2__bool DEFAULT_SCREEN_TO_FRAMEBUFFER_FUNCTION
+        internal static SWIGTYPE_p_f_r_Dali__Vector2__bool DefaultScreenToFramebufferFunction
         {
             get
             {
-                global::System.IntPtr cPtr = Interop.RenderTask.RenderTask_DEFAULT_SCREEN_TO_FRAMEBUFFER_FUNCTION_get();
+                global::System.IntPtr cPtr = Interop.RenderTask.RenderTaskDefaultScreenToFramebufferFunctionGet();
                 SWIGTYPE_p_f_r_Dali__Vector2__bool ret = (cPtr == global::System.IntPtr.Zero) ? null : new SWIGTYPE_p_f_r_Dali__Vector2__bool(cPtr, false);
                 if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
                 return ret;
             }
         }
 
-        internal static SWIGTYPE_p_f_r_Dali__Vector2__bool FULLSCREEN_FRAMEBUFFER_FUNCTION
+        internal static SWIGTYPE_p_f_r_Dali__Vector2__bool FullscreenFramebufferFunction
         {
             get
             {
-                global::System.IntPtr cPtr = Interop.RenderTask.RenderTask_FULLSCREEN_FRAMEBUFFER_FUNCTION_get();
+                global::System.IntPtr cPtr = Interop.RenderTask.RenderTaskFullscreenFramebufferFunctionGet();
                 SWIGTYPE_p_f_r_Dali__Vector2__bool ret = (cPtr == global::System.IntPtr.Zero) ? null : new SWIGTYPE_p_f_r_Dali__Vector2__bool(cPtr, false);
                 if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
                 return ret;
             }
         }
 
-        public static bool DEFAULT_EXCLUSIVE
+        public static bool DefaultExclusive
```

Comment:
could you add " [EditorBrowsable(EditorBrowsableState.Never)]" here for public API?
This class itself is hidden-API but by adding "[EditorBrowsable(EditorBrowsableState.Never)]" into each method, it would be more clear and intuitive.